checker/src/main/java/org/checkerframework/checker/resourceleak/ResourceLeakAnnotatedTypeFactory.java

```diff
@@ -70,6 +78,13 @@
    */
   private final BiMap<LocalVariableNode, Tree> tempVarToTree = HashBiMap.create();
 
+  /**
+   * Map to store a set of FieldToFinalizers for classes. Keys are class trees; values are sets of
+   * FieldToFinalizers which represents a set of method Elements that satisfy @MustCall obligation
+   * of FieldToFinalizers's field along some path to their regular exit points
+   */
+  Map<ClassTree, Set<FieldToFinalizers>> classToFieldToFinalizers = new HashMap<>();
```

Comment:
As far as I can tell this mapping is never used in this PR, only updated.  It might be useful for other inference tasks later, though.  Is that correct?  If so, maybe we should move its creation and relevant code to a later PR, where it is actually used?